adminpages/dashboard.php

```diff
@@ -244,7 +244,7 @@ function pmpro_dashboard_report_recent_members_callback() {
     						</strong>
     					</td>
     					<td><?php esc_attr_e( $auser->membership ); ?></td>
-    					<td><?php echo date_i18n( get_option( 'date_format' ), strtotime( $theuser->user_registered, current_time( 'timestamp' ) ) ); ?></td>
+    					<td><?php echo date_i18n( get_option( 'date_format' ), strtotime( get_date_from_gmt( $theuser->user_registered, 'Y-m-d H:i:s' ), current_time( 'timestamp' ) ) ); ?></td>
```

Comment:
Removing format argument that matches the default argument value.

```suggestion
    					<td><?php echo date_i18n( get_option( 'date_format' ), strtotime( get_date_from_gmt( $theuser->user_registered ), current_time( 'timestamp' ) ) ); ?></td>
```